store/store.go

```diff
@@ -180,6 +194,9 @@ func NewMongoStoreClient(config *tpMongo.Config) *MongoStoreClient {
 		log.Fatal(dataStoreAPIPrefix, fmt.Sprintf("Invalid MongoDB connection string: %s", err))
 	}
 
+	// Passed all checks Mongo configuration is valid
+	mongoConfigValid.Set(1)
```

Comment:
This gauge will always be `1` because the service won't start if the configuration is invalid - `log.Fatal` [here](https://github.com/tidepool-org/tide-whisperer/pull/90/files#diff-1f8872a5d54402aa220c723deb16293cR188) and [here](https://github.com/tidepool-org/tide-whisperer/pull/90/files#diff-1f8872a5d54402aa220c723deb16293cR194) terminates the process.

Reply:
Thanks!